chaosreader/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM komand/python-pypy3-plugin:2
+FROM komand/python-pypy3-full-plugin:3
```

Comment:
Should we be using a smaller image here the full images are very large and I'm not sure it's worth the extra 700-800MB to stick with pypy

Reply:
Here it is needed to use debian jessie. Without it docker use new version `chaosreader` app which not work with plugin. We could change it to `komand/python-3-37-slim-plugin` but it need to rewrite some code in plugin.

Reply:
@mgorny-morecode you can use this image, which is a stripped down Debian image: `komand/python-3-37-plugin:3`

Reply:
Debian stretch can't be used. In this version chaosreader was updated and isn't compatible with plugin. So here need to be used debian jessie.

Reply:
Please update the plugin to work with the latest ChaosReader package, in this instance.